docs/develop/showcase.md

```diff
@@ -0,0 +1,103 @@
+---
+id: showcase
+title: Showcase
+---
+
+import ThemedImage from '@theme/ThemedImage';
+
+An incomplete list of apps, wallets and websites that built on KILT.
+If you are missing from the list and want to be added, you are welcome to open a PR.
+
+<br/>
+<br/>
```

Comment:
maybe one `<br/>` is enough? sorry for nitpicking

Reply:
[59a314a](https://github.com/KILTprotocol/docs/pull/92/commits/59a314afa3947ff81458303576d19fa1185ae4d4) No problem! :)